sdk/loadtestservice/Azure.ResourceManager.LoadTestService/tests/ScenarioTests/LoadTestQuotaOperations.cs

```diff
@@ -0,0 +1,91 @@
+        {
+            if (Mode == RecordedTestMode.Record || Mode == RecordedTestMode.Playback)
+            {
+                await CreateCommonClient();
+            }
+            _quotaResourceCollection = Subscription.GetQuotaResources(LoadTestResourceHelper.RESOURCE_LOCATION);
+        }
+
+        [OneTimeTearDown]
+        public void Cleanup()
+        {
+            CleanupResourceGroups();
+        }
+
+        [RecordedTest]
+        [AsyncOnly]
+        public async Task GetQuotaLimitAndUsage()
+        {
+            //// Quota get limit and usage tests
+            var quotaResponse = await _quotaResourceCollection.GetAsync("maxConcurrentTestRuns");
+            Assert.IsNotNull(quotaResponse);
+            Assert.IsNotNull(quotaResponse.Value);
+            Assert.IsNotNull(quotaResponse.Value.Data);
+            Assert.IsNotNull(quotaResponse.Value.Data.Name);
+            Assert.IsNotNull(quotaResponse.Value.Data.Limit);
+            Assert.IsNotNull(quotaResponse.Value.Data.Usage);
+            Assert.AreEqual("maxConcurrentTestRuns", quotaResponse.Value.Data.Name);
+        }
+
+        [RecordedTest]
+        [AsyncOnly]
+        public async Task CheckQuotaAvailability()
+        {
+            //// Quota check availability tests
+            Response<QuotaResource> quotaResponse = await _quotaResourceCollection.GetAsync("maxConcurrentTestRuns");
+
+            Assert.IsNotNull(quotaResponse);
+            Assert.IsNotNull(quotaResponse.Value);
+            QuotaResource quotaResource = quotaResponse.Value;
+            Assert.IsNotNull(quotaResource.Data);
+            Assert.IsNotNull(quotaResource.Data.Name);
+            Assert.IsNotNull(quotaResource.Data.Limit);
+            Assert.IsNotNull(quotaResource.Data.Usage);
+            Assert.AreEqual("maxConcurrentTestRuns", quotaResource.Data.Name);
+
+            QuotaBucketRequestPropertiesDimensions dimensions = new QuotaBucketRequestPropertiesDimensions(
+                Subscription.Id.SubscriptionId, LoadTestResourceHelper.RESOURCE_LOCATION);
+
+            QuotaBucketContent quotaAvailabilityPayload = new QuotaBucketContent(
+                quotaResponse.Value.Data.Id,
+                quotaResource.Data.Name,
+                quotaResource.Data.ResourceType,
+                null,
+                quotaResource.Data.Usage,
+                quotaResource.Data.Limit,
+                quotaResource.Data.Limit,
+                dimensions);
+
+            Response<CheckQuotaAvailabilityResponse> checkAvailabilityResponse = await quotaResponse.Value.CheckAvailabilityAsync(quotaAvailabilityPayload);
+            Assert.IsNotNull(checkAvailabilityResponse);
+            Assert.IsNotNull(checkAvailabilityResponse.Value);
+            Assert.AreEqual("maxConcurrentTestRuns", checkAvailabilityResponse.Value.Name);
+            Assert.True(checkAvailabilityResponse.Value.IsAvailable.Value == true || checkAvailabilityResponse.Value.IsAvailable.Value == false);
```

Comment:
Why do we check here both true and false? This will cause this check to keep succeeding.

Reply:
The value can be either true or false.
In case of live tests, either true or false can be returned based on the state of service quota availability.
So, checking for both values. (Also NUnit package lacks Assert.IsBoolean)